ui/src/app/reports/components/reports.tsx

```diff
@@ -2,351 +2,132 @@ import {Page} from 'argo-ui/src/index';
-                        <TagsInput
-                            placeholder='Labels'
-                            tags={this.state.labels}
-                            autocomplete={this.state.autocompleteLabels}
-                            onChange={labels => this.fetchReport(this.state.namespace, labels)}
-                        />
-                    </div>
-                    <div className=' columns small-12 xlarge-12'>
-                        <p className='wf-filters-container__title'>Workflow Template</p>
-                        <DataLoaderDropdown
-                            load={() =>
-                                services.workflowTemplate
-                                    .list(this.state.namespace, [])
-                                    .then(list => list.items || [])
-                                    .then(list => list.map(x => x.metadata.name))
-                            }
-                            onChange={value => (this.workflowTemplate = value)}
-                        />
-                    </div>
-                    <div className=' columns small-12 xlarge-12'>
-                        <p className='wf-filters-container__title'>Cron Workflow</p>
-                        <DataLoaderDropdown
-                            load={() => services.cronWorkflows.list(this.state.namespace).then(list => list.map(x => x.metadata.name))}
-                            onChange={value => (this.cronWorkflow = value)}
-                        />
-                    </div>
-                    <div className=' columns small-12 xlarge-12'>
-                        <p className='wf-filters-container__title'>Phase</p>
-                        {[NODE_PHASE.SUCCEEDED, NODE_PHASE.ERROR, NODE_PHASE.FAILED].map(phase => (
-                            <div key={phase}>
-                                <label style={{marginRight: 10}}>
-                                    <input type='radio' checked={phase === this.phase} onChange={() => (this.phase = phase)} style={{marginRight: 5}} />
-                                    {phase}
-                                </label>
-                            </div>
-                        ))}
-                    </div>
+        })();
+    }, [namespace, labels]);
+
+    useCollectEvent('openedReports');
+
+    return (
+        <Page
+            title='Reports'
+            toolbar={{
+                breadcrumbs: [
+                    {title: 'Reports', path: uiUrl('reports')},
+                    {title: namespace, path: uiUrl('reports/' + namespace)}
+                ]
+            }}>
+            <div className='row'>
+                <div className='columns small-12 xlarge-2'>
+                    <ReportFilters namespace={namespace} labels={labels} onChange={onChange} />
+                </div>
+                <div className='columns small-12 xlarge-10'>
+                    <ErrorNotice error={error} />;
```

Comment:
woops just realized I copy+pasted an extra semi-colon here (from [the previous version](https://github.com/argoproj/argo-workflows/pull/11794/files#diff-471e092498589fe7d5aab61b99b2608a8eb1cd0403b1e8022d661f7a844d55a9L308)).

this actually shows up in rendering as a random semi-colon on the page, woooops 😅 

will have that fixed shortly

Reply:
Fixed in #12060